webtoolbar.py

```diff
@@ -436,6 +446,17 @@ def __switch_page_cb(self, tabbed_view, page, page_num):
     def __page_added_cb(self, notebook, child, pagenum):
         self.entry._search_popdown()
 
+    def __download_running_cb(self):
+        #print 'se llama'
+        progress = downloadmanager.overall_downloads_progress()
+        #print 'progress', progress
+        self._download_icon.update(progress)
+        if progress > 0.0:
+            self._download_icon.set_tooltip("Downloading.. " + str(progress*100) + "%")
```

Comment:
Please run this string through gettext.  Also, please use a format string (better for translators).

Also, can you display the progress without the `.0`?  Just like `72%` not `72.0%`?